wiki/osu!tourney/osu!lazer_tournament_client/en.md

```diff
@@ -0,0 +1,66 @@
+# osu!lazer tournament client
+
+**osu!lazer tournament client** is the official client that overlays osu!tourney.
```

Comment:
putting "osu!lazer tournament client" at the beginning makes it sound like a title when it's not

also the description doesn't really say anything. what does it do (u already got this a little with "overlays osu!tourney")? how does it compare to using the stable tourney client alone?